app/imports/parser/functions.js

```diff
@@ -106,7 +106,7 @@ export default {
     fn: function tableLookup(arrayNode, number){
       for(let i in arrayNode.values){
         let node = arrayNode.values[i];
-        if (node.value > number) return i;
+        if (node.value > number) return Number(i);
```

Comment:
Iinm the convention in existing code is to use `+i` instead of `Number(i)`, maybe change for consistency? Result is the same though.

Reply:
I only made it like this, cuz it explicitly states the expected type of return (sorry, I really appreciate a good typed code)

Reply:
@KatrinaKitten i used the convention you mentioned